Makefile

```diff
@@ -19,7 +24,7 @@ endif
 lint: lintdeps ## to lint the files
 	$(LINTER) run --config=.golangci-lint.yml ./...
 
-build: dep ## Build the binary file
```

Comment:
Why do we need to remove this?

Reply:
When you want to build the Dockerfile, If you run `make build` it always does a `go mod tidy` and `go build` at the same time - so you couldn't have your deps cached. with this, you can have more efficient docker builds. meanwhile by running `make` it will do `dep` and `build` (still easy for dev).